src/SnapshotRepository.php

```diff
@@ -19,7 +19,10 @@ public function getAll(): Collection
             ->filter(function (string $fileName) {
                 $pathinfo = pathinfo($fileName);
 
-                if ($pathinfo['extension'] === 'gz') {
+                if (
+                    array_key_exists('extension', $pathinfo)
```

Comment:
Could you use the null coalescing operator here?

Reply:
@freekmurze sure!